actors/builtin/multisig/multisig_actor.go

```diff
@@ -175,18 +174,12 @@ func (a *MultiSigActor) _rtApproveTransactionOrAbort(rt Runtime, callerAddr addr
 
 	expirationExceeded := (rt.CurrEpoch() > txn.Expiration)
 	if expirationExceeded {
+		// TODO: delete from state? https://github.com/filecoin-project/specs-actors/issues/5
 		rt.AbortStateMsg("Transaction expiration exceeded")
-
-		TODO()
-		// Determine what to do about state accumulation over time.
-		// Cannot rely on proposer to delete unexecuted transactions;
-		// there is no incentive (in fact, this costs gas).
-		// Could potentially amortize cost of cleanup via Cron.
 	}
 
-	AssertMsg(callerAddr.Protocol() == addr.ID, "caller address does not have ID")
 	actorID, err := addr.IDFromAddress(callerAddr)
-	autil.Assert(err == nil)
+	autil.AssertNoError(err)
```

Comment:
Note: this will conflict with https://github.com/filecoin-project/specs-actors/pull/6